src/Modules/Guilds/Wrappers/ProposalCardWrapper.tsx

```diff
@@ -44,9 +50,9 @@ const ProposalCardWrapper: React.FC<ProposalCardWrapperProps> = ({
         proposalId ? `/${chainName}/${guildId}/proposal/${proposalId}` : null
       }
       statusProps={{
-        timeDetail: proposal?.timeDetail,
         status,
-        endTime: proposal?.endTime,
+        endTimeMoment,

```

Comment:
This is just a personal taste. But I like to re-arrange the props so that you get the syntax: `endTime.moment` and `endTime.detail` thus keeping the original prop and having the value be an object instead. Something like:

`endTime: { detail: value, moment: value }`. But just a personal preference so that you don't end up with a lot of props :)